src/compiletest/runtest.rs

```diff
@@ -1013,8 +1013,8 @@ fn check_expected_errors(revision: Option<&str>,
         expected_errors.iter()
                         .fold((false, false),
                               |(acc_help, acc_note), ee|
-                                  (acc_help || ee.kind == "help:", acc_note ||
-                                   ee.kind == "note:"));
+                                  (acc_help || ee.kind == "help:" || ee.kind == "help",
+                                   acc_note || ee.kind == "note:" || ee.kind == "note"));
```

Comment:
This should  make the opt-in work for `note:` and `help:` as well, but it is still opt-in and notes and help messages are only checked in files with `help:` and `note:` annotations.

When making this changes, it seems like I overlooked `note:` and `help:` (and assumed there is more normalization going on)